Orm/Xtensive.Orm/Sql/SqlConnection.cs

```diff
@@ -67,28 +69,34 @@ public ConnectionInfo ConnectionInfo
     /// </summary>
     public int? CommandTimeout
     {
-      get { return commandTimeout; }
+      get => commandTimeout;
       set {
-        if (value!=null)
-          ArgumentValidator.EnsureArgumentIsInRange(value.Value, 0, 65535, "value");
+        if (value!=null) {
+          ArgumentValidator.EnsureArgumentIsInRange(value.Value, 0, 65535, nameof(value));
+        }
+        EnsureIsNotDisposed();
+
         commandTimeout = value;
       }
     }
 
     /// <summary>
     /// Gets the state of the connection.
     /// </summary>
-    public ConnectionState State { get { return isDisposed ? ConnectionState.Closed : UnderlyingConnection.State; } }
-    
+    public ConnectionState State => isDisposed ? ConnectionState.Closed : UnderlyingConnection.State;
+
     /// <summary>
     /// Creates and returns a <see cref="DbCommand"/> object associated with the current connection.
     /// </summary>
     /// <returns>Created command.</returns>
     public DbCommand CreateCommand()
     {
+      EnsureIsNotDisposed();
       var command = CreateNativeCommand();
-      if (commandTimeout!=null)
+      if (commandTimeout!=null) {
         command.CommandTimeout = commandTimeout.Value;
+      }

```

Comment:
Strangely, you still have editorconfig settings not applied sometimes

Reply:
Fixed